pkg/skaffold/util/util.go

```diff
@@ -67,6 +67,16 @@ func StrSliceContains(sl []string, s string) bool {
 	return false
 }
 
+// Tests if a file exists
+func FileExists(filapath string) bool {
```

Comment:
we should propagate the error if there is error os.Stat-ing the file, instead of swallowing it to `false`